assets/scrips/index.js

```diff
@@ -65,16 +65,20 @@ function randomArrayElement(array) {
  */
 class Card {
```

Comment:
This class kill all SOLID principles JAJAJAJAJA. It has a lot of coupling but it's really difficult of seeing. If you are interested please contact me, I will explain you what will be the problem with this `class` if you wanna upgrade it

Reply:
Please you must click on "merge button" so I will be sure you have read this comment

Reply:
Ok. I'm always opened to doing things better. I'll contact you.